### DISCUSSION TITLE
Looking for a gruvbox theme for Zettlr...

### DISCUSSION THREAD

**Discussion by Valdus92** [BryanJenksCommunity/FAQ#191] [Q&A] [answered]
I like to call it Inspiration in Brown. I have been trying to find a theme that works with zettlr with no luck. 

Any suggestions gruvbox King?

**Comment by tallguyjenks** [comment, marked as answer]
i dont use zettlr so 🤷🏻‍♂️

**Comment by Valdus92** [comment]
Not a problem, I seem to have moved to logseq. Your theme for that one is groovy but the editor pane is rather dark. 

Thanks
